staging/src/k8s.io/client-go/util/flowcontrol/throttle.go

```diff
@@ -31,8 +31,6 @@ type RateLimiter interface {
 	TryAccept() bool
 	// Accept returns once a token becomes available.
 	Accept()
-	// Stop stops the rate limiter, subsequent calls to CanAccept will return false
-	Stop()
```

Comment:
Removing this would break the public interface, which we do actually care about in client go.